src/helpers/supportsLSPInterface/supportsLSPInterface.ts

```diff
@@ -0,0 +1,42 @@
+import { BytesLike, Provider, isAddress, isAddressable } from 'ethers';
+import { ERC165, ERC165__factory, InterfaceId, InterfaceIdName } from '../..';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+
+export async function supportsLSPInterface(
+    interfaceId: InterfaceId | InterfaceIdName,
+    contract: ERC165,
+): Promise<boolean>;
+export async function supportsLSPInterface(
+    interfaceId: InterfaceId | InterfaceIdName,
+    contract: ERC165 | BytesLike,
+    provider?: Provider,
+): Promise<boolean> {
+    if (!isAddress(contract) && !isAddressable(contract)) {
+        throw new Error(
+            `The parameter \`contract\` is not a valid address nor a valid contract instance of ERC165 contract. Value: '${contract}'`,
+        );
+    }
+
+    let contractInstance: ERC165;
+
+    if (isAddress(contract)) {
```

Comment:
If the two conditionals line 14 are skipped, it means that this line will be sure to pass.

Reply:
Yep, here I need to know which of those condition is true